shimmer/develop/what-is-shimmer/layer-1-landscape.md

```diff
@@ -0,0 +1,141 @@
+| [Tokenization](#tokenization)                                  | None                                                                                                   | NFTs, Native Tokens, Alias Outputs                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| [Storage deposits](#storage-deposits)                          | None                                                                                                   | Yes                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| [Output unlock conditions](#output-unlock-conditions)          | [Address](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/) | [Address](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/), [Storage Deposit Returns](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#storage-deposit-return), [Expiration](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#expiration), [Timelock](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/unlock_conditions/#timelock) |
+| [IOTA Smart Contract (ISC)](#iota-smart-contracts-isc) Support | No                                                                                                     | On a [private Tangle](https://wiki.iota.org/shimmer/smart-contracts/guide/development_tools/docker_preconfigured/)                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
On a private tangle is not correct. Technically you could run it everywhere. Can we replace it with something like: In alpha/beta state?

Reply:
Alpha till the beta is announced?